src/core/src/main/java/org/apache/jmeter/report/dashboard/HtmlTemplateExporter.java

```diff
@@ -62,11 +62,8 @@
 public class HtmlTemplateExporter extends AbstractDataExporter {
     private static final FileFilter HTML_REPORT_FILE_FILTER =
         file ->
-            (file.isFile() &&
-                    "index.html".equals(file.getName()))
-                    || (file.isDirectory() &&
-                            ("content".equals(file.getName()) ||
-                                    file.getName().startsWith("sbadmin2-")));
+            (file.isFile() && "index.html".equals(file.getName()))
```

Comment:
Reading this after a long time, I wonder if it would be better to extract the conditions into a small function.